packages/angular_devkit/benchmark/src/interfaces.ts

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
nit: should this be a `.d.ts` file instead to enforce that it has no values?

Reply:
Done